README.md

```diff
@@ -178,6 +178,27 @@ If you need to send data to Jean-Zay a good idea is to use `rsync`. E.g.:
 rsync -avz -e ssh --progress  user@jeanzay:/gpfsscratch/your/remote/dir/ /your/local/database/
 ```
 
+## How to use interactive mode
+
+Interactive mode using SLURM  can be done by using two commands:
```

Comment:
In an ideal world this would be explained in the official doc: http://www.idris.fr/eng/jean-zay/gpu/jean-zay-gpu-exec_interactif-eng.html.

I find the official doc unnecessarily complicated if I am being honest. I tried to tell them but it did not have an impact yet.

 I find this works for me to get an interactive job:
```
srun --ntasks=1 --gres=gpu:1 --pty bash -i
```

Reply:
Ok, thanks, @lesteve  
I edited the doc and added a single-line command (similar to the command that you use).

Reply:
I pushed a minor tweak, merging thanks!